app/controllers/renewal_letter_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class RenewalLetterController < ApplicationController
+  def show
+    registration = WasteExemptionsEngine::Registration.find(params[:id])
+    authorize! :read, registration
+
+    show_as_html = params[:show_as_html].present? && params[:show_as_html] == "true"
```

Comment:
```suggestion
    show_as_html = params[:show_as_html] == "true"
```
I don't think we need the extra `present?` here :)

Reply:
Resolved by https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/357/commits/5ca1395696143a211817bd537f9f3e9441fbad31 (and then some)